src/connectorgen/main.go

```diff
@@ -16,13 +16,13 @@ package main
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"os"
 	"slices"
 	"strings"
 	"time"
 
-	"github.com/goccy/go-json"
```

Comment:
Nitpick: it would be good to keep `goccy/go-json` (it was maybe an unintentional change), since we might have a large list of connectors, so we kinda care about performance.